extensions/amp-fx-flying-carpet/0.1/amp-fx-flying-carpet.js

```diff
@@ -143,9 +143,28 @@ export class AmpFlyingCarpet extends AMP.BaseElement {
       this.children_
     );
     this.observeNewChildren_();
+    this.forceClipDraw_();
     return Promise.resolve();
   }
 
+  /**
+   * Something causes browsers to forget to redraw the content of the container when they become visible.
+   * @private
+   */
+  forceClipDraw_() {
+    const inob = new this.win.IntersectionObserver(
+      (entries) => {
+        const last = entries[entries.length - 1];
+        this.container_.classList.toggle(
+          'i-amphtml-fx-flying-carpet-container-fix',
+          last.isIntersecting
+        );
+      },
+      {threshold: 0.01}
```

Comment:
What is the threshold for?

Reply:
It helps for the redraw if the elements is a bit further into the page. Without, it was inconsistent and sometimes didn't fix the bug.